corehq/apps/hqadmin/management/commands/stale_data_in_es.py

```diff
@@ -73,14 +83,23 @@ def handle(self, domain, data_models, **options):
         if run_config.domain is ALL_SQL_DOMAINS:
             print('Running for all SQL domains (and excluding Couch domains!)', file=sys.stderr)
 
+        self.print_data_row(HEADER_ROW)
```

Comment:
there were just no headers before this change, yeah?

Reply:
Yeah there were no headers before, and I thought it was a nice to have. For programatic use a header doesn't matter, but I wanted to better support the workflow of pasting to a google sheet, analyzing, linking to the google sheet from a support ticket, iterating, and then eventually using the same file programmatically to pass to `republish_doc_changes`